src/components/PersonalSettings.vue

```diff
@@ -0,0 +1,264 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<div id="richdocuments" class="section">
+		<h2>
+			{{ t('richdocuments', 'Nextcloud Office') }}
+		</h2>
+
+		<span id="documents-admin-msg" class="msg" />
+
+		<!-- Template folder selection -->
+		<p>
+			<label for="templateInputField">
+				{{ t('richdocuments', 'Select a template directory') }}
+			</label><br>
+			<input id="templateInputField"
+				type="text"
+				name="templateInputField"
+				:value="templateFolder"
+				disabled>
+			<button id="templateSelectButton"
+				@click="onTemplateSelectButtonClick">
+				<span class="icon-folder"
+					:title="t('richdocuments', 'Select a personal template folder')"
+					data-toggle="tooltip" />
+			</button>
+			<button id="templateResetButton"
+				@click="resetTemplate">
+				<span class="icon-delete"
+					:title="t('richdocuments', 'Remove personal template folder')"
+					data-toggle="tooltip" />
+			</button>
+		</p>
+		<p>
+			<em>
+				{{ t('richdocuments', 'Templates inside of this directory will be added to the template selector of Nextcloud Office.') }}
+			</em>
+		</p>
+
+		<!-- Zotero -->
+		<p><strong>{{ t('richdocuments', 'Zotero') }}</strong></p>
+		<template v-if="hasZoteroSupport">
+			<div class="input-wrapper">
+				<p>
+					<label for="zoteroAPIKeyField">
+						{{ t('richdocuments', 'Enter Zotero API Key') }}
+					</label><br>
+					<input id="zoteroAPIKeyField"
+						v-model="zoteroAPIKey"
+						type="text"
+						name="zoteroAPIKeyField">
+					<button id="zoteroAPIKeySave"
+						@click="saveZoteroAPIKey">
+						<span :title="t('richdocuments', 'Save Zotero API key')" data-toggle="tooltip">
+							Save
+						</span>
+					</button>
+					<button id="zoteroAPIKeyRemove"
+						@click="resetZoteroAPI">
+						<span class="icon-delete"
+							:title="t('richdocuments', 'Remove Zotero API Key')"
+							data-toggle="tooltip" />
+					</button>
```

Comment:
Please also use NcButton and [NcTextField](https://nextcloud-vue-components.netlify.app/#/Components/NcFields?id=nctextfield) as components